### DISCUSSION TITLE
关于新GUI的WebView2管理器占用问题

### DISCUSSION THREAD

**Discussion by Xinli-GUMP** [clash-verge-rev/clash-verge-rev#2247] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/ebbb2e7d-3c2b-4df3-b92e-3e905f62d877)

为什么即使不打开GUI面板，WebVIew2也会占用不少的资源啊，100+M的内存，甚至还一直有GPU的占用，明明没有在绘制任何图像....
这是为什么？

**Comment by ShenHongFei** [comment]
https://github.com/clash-verge-rev/clash-verge-rev/discussions/2290
我提的这个可能和楼主重复了

**Reply by Xinli-GUMP** [reply]
算是一个补充吧，1.x版本虽然gui面板打开偏慢，但是占用也小了很多，这次gui框架的升级有点得不偿失。

**Reply by ShenHongFei** [reply]
说得对，不改回去我就一直用 1.x 版本了

**Comment by iwalke** [comment]
![image](https://github.com/user-attachments/assets/4d1a179f-a09a-4586-92ee-5d077fe56ab1)
https://github.com/clash-verge-rev/clash-verge-rev/releases/tag/v2.0.2

**Reply by ShenHongFei** [reply]
内存倒是真无所谓，但是 CPU 一直占着不行

**Reply by iwalke** [reply]
估计是渲染？在意的话就用回 v1.7.7 吧

**Comment by CAESIUS-TIM** [comment]
首页右上角有个“轻量模式”的按钮
![轻量模式](https://github.com/user-attachments/assets/4344b2d8-3fcd-486f-a9a6-7c71844da9ec)
一般模式
![normal](https://github.com/user-attachments/assets/d514f225-297d-4014-a2e8-6eca3e4b7951)
轻量模式
![light](https://github.com/user-attachments/assets/67e914e5-2cf9-4021-8ccd-b3c047b7a268)

---

托盘也有，但是启动后并没有✅（打勾）的提示标志
![tray](https://github.com/user-attachments/assets/dc24f740-2d4f-4b2b-8910-e966c4d01ae7)

**Reply by ShenHongFei** [reply]
终于改了吗？等有空升级下试试，谢谢

**Reply by CAESIUS-TIM** [reply]
而且可以设置自动进入轻量模式
![image](https://github.com/user-attachments/assets/d997a45d-84d7-48c9-a00e-a20fb0b6f1dd)

**Reply by ShenHongFei** [reply]
不错

**Reply by CAESIUS-TIM** [reply]
但是静默启动好像有bug #3469

**Reply by ShenHongFei** [reply]
试了下最新版，好像能用，但是有一次自动进入轻量模式没有生效，再观察观察

**Comment by Ahaohaohao** [comment, marked as answer]
换框架了

**Reply by ShenHongFei** [reply]
第一次启动时的窗口，关闭了还在，自动进入轻量模式没有生效，需要手动点轻量模式，后续才可以

**Reply by ShenHongFei** [reply]
刚更新了 05.21 最新版的
